bin/yauml

```diff
@@ -19,9 +19,10 @@
 #                  http://www.gnu.org/licenses/                                     *
 #************************************************************************************
 
-import sys, yaml, getopt, re, subprocess
+import os, sys, yaml, getopt, re, subprocess
 
-template_filename = '../templates/default.dot'
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+TEMPLATE_FILENAME = CURRENT_DIR + '/../templates/default.dot'
```

Comment:
I was not clear. What I meant by "unconventional installation" should have been "installation in unconventional directory". I just think that someone could also install YAUML in any directory, and then set his/her ``PATH`` environment variable to make it work.